CrossFAB/Controls/FloatingMenu.xaml.cs

```diff
@@ -119,7 +119,7 @@ void ArrangeChildren(object sender, EventArgs evt)
                 Children[i].Scale = 0.7;
                 AbsoluteLayout.SetLayoutBounds(Children[i], new Rectangle(0, (60 * i), 60, 60));
                 Children[i].Rotation = 180;
-                ((FloatingButton)Children[i]).ExtraCommand = new Command(() => { Collapse(AnimationTime); });
+                ((FloatingButton)Children[i]).ExtraCommand = new Command(() => {  MainButton?.OnClickCommand.Execute(null); });
```

Comment:
Have you tested this? As far as I remember, if you remove `Collapse(AnimationTime)` the menu won't collapse when a child is clicked.
Please re-add `Collapse(AnimationTime)` and confirm that it keeps working as intended.